File: IdeaProjects/Meeting System/src/models/Meeting.java
package models;

import java.util.List;

public class Meeting {
    private int id;
    private MeetingRoom room;
    private String subject;
    private Interval interval;
    private List<User> attendees;
    private User sender;

    public int getId() {
        return id;
    }

    public MeetingRoom getRoom() {
        return room;
    }

    public String getSubject() {
        return subject;
    }

    public Interval getInterval() {
        return interval;
    }

    public List<User> getAttendees() {
        return attendees;
    }

    public User getSender() {
        return sender;
    }

    public void setId(int id) {
        this.id = id;
    }

    public void setAttendees(List<User> attendees) {
        this.attendees = attendees;
    }

    public void setInterval(Interval interval) {
        this.interval = interval;
    }

    public void setRoom(MeetingRoom room) {
        this.room = room;
    }

    public void setSender(User sender) {
        this.sender = sender;
    }

    public void setSubject(String subject) {
        this.subject = subject;
    }


    public void addInMeeting(List<User> person){}

    public void removeFromMeeting(User person){}
}

File: 09_Design Problems/04_Meeting Schedular System/Models/Meeting.cpp
#include <iostream>
#include <list>
#include "User.cpp"
#include "MeetingRoom.cpp"
#include "Invterval.cpp"

using namespace std;

class Meeting {
    int id;
    MeetingRoom room;
    string subject;
    Interval interval;
    list<User> attendees;
    User sender;

    public:
        void addInMeeting(list<User> attendees) {

        }   

        void removeFromMeeting(User user) {

        }
};